serverlessworkflow/modules/ROOT/pages/eventing/consume-produce-events-with-knative-eventing.adoc

```diff
@@ -1 +1,290 @@
-//= Consuming and producing events on Knative Eventing
\ No newline at end of file
+= Consuming and producing events on Knative Eventing
+:compat-mode!:
+// Metadata:
+:description: Consuming and producing events on Knative Eventing
+:keywords: kogito, workflow, serverless, events, knative, cloudevents
+// links
+:knative_eventing_doc_url: https://knative.dev/docs/eventing/
+:knative_serving_services_url: https://knative.dev/docs/serving/services/
+:kubernetes_probes_url: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/
+:quarkus_k8s_url: https://quarkus.io/guides/deploying-to-kubernetes
+
+This document describes how you can configure Serverless Workflow to interact with link:{knative_eventing_doc_url}[Knative Eventing]. 
+
+Knative Eventing abstracts the events consumption via the link:{knative_eventing_doc_url}/sources[Source] and link:{knative_eventing_doc_url}/sinks[Sink] components. An event source is the Kubernetes object that produces the event. The sink is the Kubernetes object that receives the event. The workflow application can act as a source, a sink or both in the Knative Eventing platform.
+
+You need to add the Kogito Knative Eventing add-on dependency to indicate that you are using Knative Eventing. To enable Serverless Workflow to use Knative Eventing, add the following dependency to the `pom.xml` file of your project if using Maven:
+
+.Add dependency for Kogito Knative Eventing add-on in `pom.xml`
+[source,json]
+----
+<dependency>
+    <groupId>org.kie.kogito</groupId>
+    <artifactId>kogito-addons-quarkus-knative-eventing</artifactId>
+</dependency>
+----
+
+This add-on will take care of the required dependencies and any additional configurations that the workflow application needs to interact with the Knative Eventing platform.
+
+Although the default configuration that the Quarkus Knative Eventing add-on provides ought to be enough for the majority of uses cases, sometimes you may need to do additional configuration to serve a specific scenario.
+
+[[con-knative-eventing-add-on-source-configuration]]
+== Knative Eventing add-on Source configuration
+
+The following properties are only valid if the workflow has at least one link:{spec_doc_url}#Event-Definition[event definition] of the type `produced`. That's when the workflow application produces events, acting as a Knative Source.
```

Comment:
these properties are the ones starting on line 44?
if so, ins't better to move this paragraph before the properties section?